main.cc

```diff
@@ -1056,6 +1056,13 @@ int main(int ac, char** av) {
                 view_builder.invoke_on_all(&db::view::view_builder::start).get();
             }
 
+            // Truncate `clients' CF - this table should not persist between server restarts.
```

Comment:
Please wrap it into a function with a self describing name to hide the complexity in this already quite long function.